MathlibTest/Isolate.lean

```diff
@@ -0,0 +1,67 @@
+/-
+Copyright (c) 2025 Heather Macbeth. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Heather Macbeth
+-/
```

Comment:
Could we have another test file with an `import Mathlib`, and a test or two for the log and sqrt tags you've added?